frontend-wasm/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "midenc-frontend-wasm"
-description = "Build MidenIR from Wasm"
-version.workspace = true
+description = "Build Miden HIR from Wasm"
```

Comment:
```suggestion
description = "Wasm frontend for the Miden compiler"
```

Reply:
I'm less opinionated on this one, if you prefer the current phrasing, feel free to ignore this suggestion.